Packs/Trello/Integrations/Trello/Trello.yml

```
@@ -8,38 +11,45 @@ configuration:
   name: url
   required: true
   type: 0
+  section: Connect
 - display: Fetch incidents
   name: isFetch
   required: false
   type: 8
+  section: Collect
 - display: Use system proxy settings
   name: proxy
   required: false
   type: 8
-- additionalinfo: ID of Trello board used both to fetch incidents and as the default
-    for all integration commands.
+  section: Connect
+  advanced: true
+- additionalinfo: ID of Trello board used both to fetch incidents and as the default for all integration commands.
   display: Default and Fetch Board ID
   hidden: false
   name: board_id
   required: false
   type: 0
+  section: Collect
+  advanced: true
```

```suggestion
  section: Connect
  advanced: true
```
Used as default for all integration commands